tests/acceptance/features/apiSharingNg/enableDisableShareSync.feature

```diff
@@ -413,3 +413,84 @@ Feature:  enable or disable sync of incoming shares
         }
       }
       """
+
+
+  Scenario: try to disable share sync of a non-existent resource
+    When user "Brian" tries to disable share sync of a resource "nonexistent" using the Graph API
+    Then the HTTP status code should be "422"
```

Comment:
should it be 404 for non-existing items?
@grgprarup Could you open an issue for this?
```suggestion
    Then the HTTP status code should be "404"
```

CC @ScharfViktor

Reply:
Response message seems valid `invalid driveID or itemID`. But not sure about the status code.

Reply:
after talking with @rhafer -> 404 is better. plz open the issue

Reply:
@ScharfViktor issue is created https://github.com/owncloud/ocis/issues/9001